tools/appliance/build.sh

```diff
@@ -271,9 +271,9 @@ function xen_server_export() {
   local result=$?
   set -e
   if [ ${result} == 0 ]; then
-    qemu-img convert -f qcow2 -O raw "dist/${appliance}" img.raw
-    vhd-util convert -s 0 -t 1 -i img.raw -o stagefixed.vhd
-    faketime '2010-01-01' vhd-util convert -s 1 -t 2 -i stagefixed.vhd -o "${appliance_build_name}-xen.vhd"
+    cp "dist/${appliance}" "dist/${appliance}-xen"
+    vhd-util convert -s 0 -t 1 -i "dist/${appliance}-xen" -o stagefixed.vhd
+    faketime '2010-01-01' qemu-img convert -S 512 -O vpc -o subformat=dynamic stagefixed.vhd "${appliance_build_name}-xen.vhd"
```

Comment:
@PaulAngus can you confirm if this could break template on XenServer, as qemu-img is used to convert the staged vhd file to `vpc` output format that corresponds to HyperV vhd?